Dockerfile

```diff
@@ -44,7 +44,7 @@ FROM gcr.io/distroless/static-debian12:nonroot AS image
 WORKDIR /
 COPY --from=build /function /function
 
-COPY package.yaml /package.yaml
+COPY package/crossplane.yaml /package.yaml
```

Comment:
Wondering if `crossplane xpkg` does this

Reply:
Right, this metadata is an argument to `crossplane xpkg build --package-root=package/`